### ISSUE TITLE
Which part of the paper does "loss_ccl" in trainers.py correspond to，and how tothe accuracy？

### ISSUE THREAD

**Issue by hzd7117** [chenhao2345/ICE#3]
Hello, it's a great honor to read your excellent work, but when reproducing your paper code, I have the following questions: 1. Which part of the paper does "loss_ccl" in trainers.py correspond to? This loss function does not seem to improve the effect of the model. 2. At present, the highest accuracy of my reproduction is only 76.7%. How can I achieve the accuracy pointed out in the paper?

Thank you for your answer!

**Comment by hzd7117** [comment]
In addition, add the experimental environment unbuntu16.04.6, pytorch1.7, and the graphics card is Tesla V100

**Comment by chenhao2345** [comment]
Hi, loss_ccl corresponds to L_agnostic in Equation (5) of the paper. 

Our results are obtained batch size and learning rate tuned with **4 GPUs**. Please try to run our code with 4 GPUs, otherwise, you may have to tune batch size and learning rate for 1 GPU.

**Comment by hzd7117** [comment]
> 

thanks！
